<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
   padding: 5px;
}
QPushButton:focus {
    /* border: 1px dotted black; */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_build_ships">
         <attribute name="title">
          <string>Build Ships</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QComboBox" name="cmb_planets_build"/>
              </item>
              <item>
               <widget class="QPushButton" name="btn_get_shipyards_build">
                <property name="text">
                 <string>Get shipyards on this planet</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="lbl_build_ports_available">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tbl_shipyards_build">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="columnCount">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tbl_ships_build"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_build_ships">
              <property name="text">
               <string>Build!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_scuttle_ships">
         <attribute name="title">
          <string>Scuttle Ships</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="cmb_planets_scuttle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_get_ships_scuttle">
                <property name="text">
                 <string>Get ships on this planet</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="tbl_ships_scuttle">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <property name="columnCount">
               <number>5</number>
              </property>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_status">
         <attribute name="title">
          <string>Empire Status</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_get_empire_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Get Empire Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txt_empire_status">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="tabChangesFocus">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="actionChose_Config_File"/>
    <addaction name="actionChose_Config_Section"/>
    <addaction name="actionConfig_File_Status"/>
    <addaction name="separator"/>
    <addaction name="actionLog_In"/>
    <addaction name="actionLog_Out"/>
    <addaction name="separator"/>
    <addaction name="actionTest"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionClear_All_Caches"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="actionUnmute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMedia"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>status tt</string>
   </property>
   <property name="statusTip">
    <string>status st</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionChose_Config_File">
   <property name="text">
    <string>Chose Config File</string>
   </property>
   <property name="statusTip">
    <string>Select a non-default config file</string>
   </property>
  </action>
  <action name="actionChose_Config_Section">
   <property name="text">
    <string>Chose Config Section</string>
   </property>
   <property name="statusTip">
    <string>Log in using a different account</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
   <property name="statusTip">
    <string>Whoops.  I should have removed this.</string>
   </property>
  </action>
  <action name="actionLog_In">
   <property name="text">
    <string>Log In</string>
   </property>
  </action>
  <action name="actionConfig_File_Status">
   <property name="text">
    <string>Connection Status</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>App details and version information</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionClear_All_Caches">
   <property name="text">
    <string>Clear All Caches</string>
   </property>
   <property name="toolTip">
    <string>Click if you're getting old data.</string>
   </property>
   <property name="statusTip">
    <string>Click if you're getting old data.</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionUnmute">
   <property name="text">
    <string>Unmute</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_get_empire_status</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
